.github/workflows/lint-with-vale.yml

```diff
@@ -0,0 +1,21 @@
+---
+name: Linting with Vale on pull request
+on:
+  pull_request:
+    paths:
+      - 'guides/common/*.adoc'
+      - '.github/workflows/lint-with-vale.yml'
+
+jobs:
+  vale:
+    name: Linting with Vale
```

Comment:
```suggestion
    name: lint-with-vale
```

Reply:
Not sure why the name should be changed.